src/Symfony/Bridge/Twig/Command/LintCommand.php

```diff
@@ -89,7 +89,7 @@ protected function execute(InputInterface $input, OutputInterface $output): int
         $this->format = $input->getOption('format') ?? (GithubActionReporter::isGithubActionEnvironment() ? 'github' : 'txt');
 
         if (['-'] === $filenames) {
-            return $this->display($input, $output, $io, [$this->validate(file_get_contents('php://stdin'), uniqid('sf_', true))]);
+            return $this->display($input, $output, $io, [$this->validate(file_get_contents('php://stdin'), 'Standard Input')]);
```

Comment:
other alternatives would be `php://stdin` or just `stdin` instead, as we are already familiar with these stream abbreviations.

outputs e.g.
```
ERROR in php://stdin (line -1)
// or
ERROR in stdin (line -1)
```

Reply:
It's also aimed at front-end developers who only know Twig and not PHP. It could be more specific: "Input Template".

Reply:
In any case, I don't think it matters, there's only one linted template, so no doubt about it.